README.md

```diff
@@ -1,25 +1,48 @@
 # Setup
 
-Install Bitbar:
+## Fork and clone this repo
+
+Fork [`lewagon/bitbar`](https://github.com/lewagon/bitbar) to your own GitHub account and clone it on your computer:
 
 ```bash
-brew cask install bitbar
+export GITHUB_USERNAME=`gh api user | jq -r '.login'`
+cd ~/code/GITHUB_USERNAME
+gh repo clone bitbar
 ```
 
-Clone this repo on your laptop then set the plugin as executable:
+Then make the `ticket.10s.rb` file executable:
 
 ```bash
-mkdir -p ~/code/lewagon
-gh repo clone lewagon/bitbar
-cd bitbar
+cd ~/code/GITHUB_USERNAME/bitbar
```

Comment:
If you had `gh auth login -s 'user:email' -w` previously you can do:

```bash
export GITHUB_USERNAME=`gh api user | jq -r '.login'`
```

And then you can use `$GITHUB_USERNAME` in the other commands

Reply:
:ok_hand: :pray: :heavy_check_mark: